xline-client/tests/lease.rs

```diff
@@ -0,0 +1,102 @@
+use common::get_cluster_client;
+use xline_client::{
+    error::Result,
+    types::lease::{
+        LeaseGrantRequest, LeaseKeepAliveRequest, LeaseRevokeRequest, LeaseTimeToLiveRequest,
+    },
+};
+
+mod common;
+
+#[tokio::test]
+async fn grant_revoke() -> Result<()> {
```

Comment:
This test case name is meaningless. Please rename it. A good test case name can be divided into three parts: the unit of work name, the expected behavior, and the scenario. You can use the pattern that looks like `xxx_should_xxx_when_xxx` or `xxx_should_success_in_normal_paht`. For example, `lease_grant_should_failed_when_xxx` or `lease_grant_should_success_in_normal_path` . Any thought of this?@rogercloud

Reply:
done